src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineLinuxLauncher.java

```diff
@@ -113,7 +121,7 @@ protected void launch(ComputeEngineComputer computer, TaskListener listener, Ins
             SCPClient scp = conn.createSCPClient();
             String tmpDir = "/tmp";
             logInfo(computer, listener, "Copying slave.jar to: " + tmpDir);
-            scp.put(Jenkins.getInstance().getJnlpJars("slave.jar").readFully(), "slave.jar", tmpDir);
+            scp.put(Jenkins.get().getJnlpJars("slave.jar").readFully(), "slave.jar", tmpDir);
```

Comment:
Is there an easy way to rename this to something like "worker.jar"? The master/slave terminology is kind of outdated. If not, a backlog item might be good.

Reply:
At the very least I see this magic string repeated a lot throughout this class, so if it was moved to a static constant then changing it in the future would be easier even if the jar name can't be changed right now. Then all you need is one TODO where that string is defined with a github issue pointing to that.

Reply:
Actually it's also repeated in the Windows launcher as well, so perhaps define it in a more central location.

Reply:
Hmm, it's a Jenkins term. I'm not sure changing it is a good idea since most people are familiar with the "slave.jar" file/terminology Jenkins uses. Thoughts?

Reply:
I think agent or worker is better. It looks like they're trying to move this way based on changes between [this old wiki page](https://wiki.jenkins.io/pages/viewpage.action?pageId=75893612) and [this new one](https://wiki.jenkins.io/display/JENKINS/Step+by+step+guide+to+set+up+master+and+agent+machines+on+Windows).

As a plus for doing this work or just adding the ticket you have something to point to as "respecting the user" and being culturally inclusive during perf :-)

Reply:
It would be out of scope for this PR, but we might also want to consider letting the user configure the name for this jar file, and default to "worker" or "agent"

Reply:
After some research, agent.jar works as well. I've changed it thus far. As for making it some kind of shared variable, I think that will have to be part of some larger refactoring effort.